e2e/passkey/passkeyProxy.ethereum.test.ts

```diff
@@ -0,0 +1,78 @@
+import '@frequency-chain/api-augment';
+import assert from 'assert';
+import {
+  createAndFundKeypair,
+  EcdsaSignature,
+  getBlockNumber,
+  getNextEpochBlock,
+  getNonce,
+  Sr25519Signature,
+} from '../scaffolding/helpers';
+import { KeyringPair } from '@polkadot/keyring/types';
+import { ExtrinsicHelper } from '../scaffolding/extrinsicHelpers';
+import { getFundingSource } from '../scaffolding/funding';
+import { getConvertedEthereumPublicKey, getUnifiedAddress } from '../scaffolding/ethereum';
+import { createPassKeyAndSignAccount, createPassKeyCall, createPasskeyPayload } from '../scaffolding/P256';
+import { u8aToHex, u8aWrapBytes } from '@polkadot/util';
+const fundingSource = getFundingSource('passkey-proxy-ethereum');
+
+describe('Passkey Pallet Ethereum Tests', function () {
+  describe('passkey ethereum tests', function () {
+    let fundedSr25519Keys: KeyringPair;
+    let fundedEthereumKeys: KeyringPair;
+    let receiverKeys: KeyringPair;
+
+    before(async function () {
+      fundedSr25519Keys = await createAndFundKeypair(fundingSource, 300_000_000n);
+      fundedEthereumKeys = await createAndFundKeypair(fundingSource, 300_000_000n, undefined, undefined, 'ethereum');
+      receiverKeys = await createAndFundKeypair(fundingSource);
+    });
+
+    it('should transfer via passkeys with root sr25519 key into an ethereum style account', async function () {
```

Comment:
Is this test description correct? It looks like the receiving account is a regular sr25519 address